solvcon/mesh.pyx

```diff
@@ -221,6 +221,17 @@ cdef class Mesh:
             if vals[0] != vals[1]:
                 msgs.append('body(%d,%d)' % vals)
             if msgs:
+                f = open("/tmp/solvcon.debug.log", "a")
```

Comment:
Fair enough.

There is a primitive facility to separate output for each of the parallel processes.  We probably should make it a default of output.